src/databases/MOAB/CMakeLists.txt

```diff
@@ -53,31 +55,33 @@ ${VISIT_INCLUDE_DIR}/avt/Pipeline/Sources
 ${VISIT_INCLUDE_DIR}/avt/VisWindow/VisWindow
 ${VISIT_INCLUDE_DIR}/visit_vtk/full
 ${VISIT_INCLUDE_DIR}/visit_vtk/lightweight
+${VTKh_INCLUDE_DIRS}
+${VTKM_DIR}/include/vtkm-1.2
+${VTKM_DIR}/include/vtkm-1.2/vtkm/thirdparty/taotuple
 ${VTK_INCLUDE_DIRS}
 )
 
-LINK_DIRECTORIES(${VISIT_LIBRARY_DIR} ${MOAB_LIBRARY_DIR} ${HDF5_LIBRARY_DIR})
+LINK_DIRECTORIES(${VISIT_LIBRARY_DIR} )
```

Comment:
Do these changes to the link_directories and target_link_libraries work for the MOAB reader?  I thought there was some funniness with this plugin that required custom CMakeLists.txt?

Reply:
this was a mistake, thanks for catching .